.gitignore

```diff
@@ -29,3 +29,4 @@ xcuserdata/
 *.dSYM
 
 build.log
+*.DS_Store
```

Comment:
This doesn't belong here, you need to configure your global gitignore to include it. Revert.

Reply:
Where is the global gitignore?